aptos-move/framework/aptos-framework/sources/genesis.move

```diff
@@ -93,7 +94,7 @@ module aptos_framework::genesis {
             b"multi_agent_script_prologue",
             b"epilogue",
         );
-
+        nonce_validation::initialize(&aptos_framework_account);
```

Comment:
this changes a few table handles, not sure how important it is. (whether we hard code some table handle like staking in explorer or other places)

Reply:
For testnet and mainnet, it's easy to explicitly call `initialize_nonce_table` transaction once.
For devnet and forge tests, we have to initialize nonce table somewhere. Better be in genesis?

Reply:
I think you can move the initialize to later place to not affect other tables

Reply:
Moved the nonce table initialization to end of genesis.